README.md

```diff
@@ -202,14 +215,14 @@ vars:
 ``` 
 </details>
 
-## (Optional) Step 6: Orchestrate your models with Fivetran Transformations for dbt Core™
+## (Optional) Step 7: Orchestrate your models with Fivetran Transformations for dbt Core™
```

Comment:
I don't believe this is needed anymore and can be `6` as we are no longer making a separate step for these variables.

Reply:
Oops, fixed now.